### ISSUE TITLE
b_sanitize exhaustivity 

### ISSUE THREAD

**Issue by elmarco** [mesonbuild/meson#5154]
Is b_sanitize supposed to be exhaustive? If not, what is the scope?

fwiw, there are myriads of options: https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html

Some are not even on this page, like 'fuzzer', which I am interested in.

**Comment by jpakkane** [comment]
The design was originally made when there were like 4 choices. Now Clang seems to be adding them faster than we can make releases. Should we just change the option to be a freeform string instead?

**Comment by cwfitzgerald** [comment]
I think that would be beneficial, it would keep the onus of meson to constantly keep things updated.

**Comment by pwnorbitals** [comment]
Yes, i think this option should be changed to allow easier access to sanitizers, too. Has there been any recent work in this direction ?
